.github/SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+This is a short excerpt of our [security bounty program](https://yoast.com/security-program/):
+
+### The rules
+- Please provide detailed reports with reproducible steps. If the report is not detailed enough to reproduce the issue, the issue will not be eligible for a reward.
+- Submit one vulnerability per report, unless you need to chain vulnerabilities to provide impact.
+- For duplicates, we only award the first report that was received (provided that it can be fully reproduced).
+- Multiple vulnerabilities caused by one underlying issue will only be eligible for one reward.
+- When testing has an overlap with systems or services not owned by you, the tester, make a good faith effort to avoid privacy violations, destruction of data, and interruption or degradation of that service. Only interact with accounts you own or with the explicit permission of the account holder.
+
+### Disclosure Policy
+Please do not discuss any vulnerabilities (even resolved ones) without express consent.
+
+### Submit your report
+When you've found a security issue that abides by the rules and scope of this project, please submit the report to us via security@yoast.com. In your mail, make sure to include:
+
+- The calculation of the CVSS (using the CVSS calculator)
+- The impact of the issue
+- A detailed guide on how to reproduce the issue
+- Provide the email address you used to create a MyYoast-account (if applicable)
```

Comment:
I'd propose starting all these lines with a lowercase character and ending them with a semi-colon, as you started the list with "make sure to include".